packages/integration-tests/tests/retrospective-demo/smoke.test.ts

```diff
@@ -1,40 +0,0 @@
-import config from '../config'
```

Comment:
This smoke test was mainly useful in the prototype. We have coverage of this with our new tests now.